docs/content/v0.20/packaging.md

```diff
@@ -0,0 +1,28 @@
+# Packaging and Delivery
+
+## status-addon
+
+The [status-addon](https://github.ibm.com/dettori/status-addon) repo is the source of a RedHat-style operator. 
+
+The operator is delivered by a Helm chart at `quay.io/pdettori/status-addon-chart`. Chart versioning? How is the chart built, pushed?
+
+There are probably one or more container images involved.
+
+## KubeStellar
+
+### Central container image
+
+Appears to be at `ghcr.io/kubestellar/kubestellar/kubestellar-operator:0.20.0-alpha.1`. `make ko-build-push` will create that image. `make ko-build-local` will make a local image for just the local platform.
+
+### KubeStellar Operator Helm Chart
+
+The source for a Helm chart is in [core-helm-chart](../../../core-helm-chart). `make chart` (re)derives it from loal sources.
+
+This chart creates (among other things) a `Deployment` object that runs a container from the image `ghcr.io/kubestellar/kubestellar/kubestellar-operator:0.20.0-alpha.1`.
+
+### KubeFlex PostCreateHooks
+
+There are two `PostCreateHook` objects defined in [config/postcreate-hooks](../../../config/postcreate-hooks).
+
+- `ocm.yaml` adds `clusteradm` by running a container using the image `quay.io/kubestellar/clusteradm:0.7.1` which is built from ... what?
```

Comment:
It's built from https://github.com/open-cluster-management-io/clusteradm - since it's a CLI there are no community images available for it so we are building our own. @francostellari submitted https://github.com/kubestellar/kubestellar/pull/1602 to add the build script for this.

Reply:
thanks, noted.